app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -153,12 +153,12 @@ def scan_job_create(entity)
     check_policy_prevent(:request_containerimage_scan, entity, :raw_scan_job_create)
   end
 
-  def raw_scan_job_create(entity)
+  def raw_scan_job_create(target_class, target_id)
```

Comment:
perhaps name the arg "target_class_name".
YMMV but when I hear "class" I always expect a class object.